File: app/src/main/java/dev/leonlatsch/kolibri/broker/MessageRecyclerChangeListener.java
package dev.leonlatsch.kolibri.broker;

import dev.leonlatsch.kolibri.database.model.Message;

/**
 * Listener for the MessageRecycler
 *
 * @author Leon Latsch
 * @since 1.0.0
 */
public interface MessageRecyclerChangeListener {

    /**
     * Called when a message has changed or a new is received
     *
     * @param message
     */
    void receive(Message message);
}

File: app/src/main/kotlin/dev/leonlatsch/kolibri/broker/MessageRecyclerChangeListener.kt
package dev.leonlatsch.kolibri.broker

import dev.leonlatsch.kolibri.database.model.Message

/**
 * Listener for the MessageRecycler
 *
 * @author Leon Latsch
 * @since 1.0.0
 */
interface MessageRecyclerChangeListener {

    /**
     * Called when a message has changed or a new is received
     *
     * @param message
     */
    fun receive(message: Message)
}
